crates/ra_hir/src/semantics.rs

```diff
@@ -451,6 +451,10 @@ impl<'a, DB: HirDatabase> SemanticsScope<'a, DB> {
     pub fn resolve_hir_path(&self, path: &Path) -> Option<PathResolution> {
         resolve_hir_path(self.db, &self.resolver, path)
     }
+
+    pub fn resolve_hir_path_qualifier(&self, path: &Path) -> Option<PathResolution> {
```

Comment:
could you add a doc comment here? something like the one for the function in `source_analyzer`.